doc/manuals/admin/database_model.md

```diff
@@ -303,8 +302,7 @@ Fields:
     notification sent associated to a given subscription.
 -   **count**: the number of notifications sent associated to
     the subscription.
--   **format**: the format to use to send notifications, either "XML"
-    or "JSON".
+-   **format**: the format to use to send notifications, "JSON".
```

Comment:
Please use the following:

"the format to use to send notification, either "XML" or "JSON". However, note that XML has been deprecated in Orion 0.23.0 and that this field eventually will be removed"